app/templates/network/show.hbs

```diff
@@ -1,84 +1,92 @@
 {{!-- Sidebar will stay in the main outlet, everything
   else will stay in sidebar's outlet --}}
 
-<div id="sidebar-toggle" {{action "openSidebar"}}>
-  <i class="fa fa-arrow-circle-right"></i>
-</div>
-
 
-{{! @TODO: create a "tutorial" component containing all #legend markup
+{{! @TODO: create a "tutorial" component/paartial containing all #legend markup
     in this template, show "tutorial" by default. hide tutorial if user clicks "close" button
     also, no other interaction with network, sidebar or network options should be allowed if tutorial is displayed.
 }}
-<section id="legend">
+<section id="legend" class="network-loading-{{stabilizationPercent}} {{if networkLoaded 'network-loaded network-just-loaded'}}">
 
-  <div class="loading-bar">
-    {{progress-bar percent=stabilizationPercent message="Please wait, the network is being loaded..."}}
-  </div>
+  <div class="legend-wrapper">
 
-  <div class="launch-network launch-network{{stabilizationPercent}}">
-    Your Network is loaded!
-    <a class="btn-tiny transparent primary" {{action "toggleInfo" on="click"}}>
-      Launch {{fa-icon "arrow-circle-right"}}
-    </a>
-  </div>
+    {{! Shows when the network is loading and cannot be yet accessed }}
+    <div class="network-loader">
+      <span>Hold on, your network is loading..{{stabilizationPercent}}.</span>
```

Comment:
You forgot to add a `%` sign here.

Reply:
Also there are only two dots!

Reply:
oops, that was just for testing purposes. thanks for noticing.